File: WinFormsApplication/Dto/FilmDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace hltb.Dto
{
    public class FilmDto : ISearchable
    {
        [JsonPropertyName("id")]
        public long Id { get; set; }


        [JsonPropertyName("title")]
        public string Title { get; set; }

        [JsonPropertyName("rus_title")]
        public string RusTitle { get; set; }


        [JsonPropertyName("link_url")]
        public string LinkUrl { get; set; }


        [JsonPropertyName("time")]
        public long Time { get; set; }


        [JsonPropertyName("date_release")]
        public string DateRelease { get; set; }


        [JsonPropertyName("global_score")]
        public float GlobalScore { get; set; }
    }
}

File: server/backend-app/src/main/java/com/sawwere/titlecounter/backend/app/dto/film/FilmDto.java
package com.sawwere.titlecounter.backend.app.dto.film;

import com.fasterxml.jackson.annotation.JsonProperty;
import jakarta.validation.constraints.Min;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.*;

import java.time.LocalDate;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class FilmDto {
    private long id;

    @NotBlank
    private String title;

    @JsonProperty(value = "rus_title")
    private String rusTitle;

    @JsonProperty(value = "link_url")
    private String linkUrl;

    @NotNull
    @Min(0)
    @JsonProperty(value = "time")
    private Long time;

    @JsonProperty(value = "date_release")
    private LocalDate dateRelease;

    @NotNull
    @Min(0)
    @JsonProperty(value = "global_score")
    private Float globalScore;
}
